doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con-timer.adoc

```diff
@@ -3,6 +3,21 @@
 
 You can use timers to trigger logic after a certain period or to repeat specific actions at regular intervals. Timers wait for a predefined amount of time before triggering once or repeatedly.
 
+== Selecting the correct timer for your environment
+{PRODUCT} supports two types of timers:
+
+* Quartz: Recommended for use on Spring Boot and Tomcat
+* EJB: Recommended for use with {EAP}, both on-premise and {OPENSHIFT}
+
+[NOTE]
+====
+Do not use timers for the following business strategies:
+
+* Timers as your process polling strategy. For example, instead of directly calling the external service and adding a 1 second timer, use {FUSE} to register an asynchronous route. Use the {FUSE} callback to fire an event for the process to move on once the expected response is received. Create a work item handler as part of your business strategy outside of your Business Process Model and Notation (BPMN) process model and move the timer to the work item handler.
```

Comment:
Because it's the negative, I would state 'Do not' again. Do not use timer as your... Do not use timers to enforce safe...

Reply:
Done